tests/JsonSchemaMapper.Tests/TestTypes.cs

```diff
@@ -84,14 +93,18 @@ public static IEnumerable<ITestData> GetTestDataCore()
 
         // Enum types
         yield return new TestData<IntEnum>(IntEnum.A, ExpectedJsonSchema: """{"type":"integer"}""");
-        yield return new TestData<StringEnum>(StringEnum.A, ExpectedJsonSchema: """{"type": "string", "enum": ["A","B","C"]}""");
-        yield return new TestData<FlagsStringEnum>(FlagsStringEnum.A, ExpectedJsonSchema: """{"type":"string"}""");
+        // `type` is redundant here since `enum` is exclusive
+		yield return new TestData<StringEnum>(StringEnum.A, ExpectedJsonSchema: """{"type": "string", "enum": ["A","B","C"]}""");
+		// why not use `enum`?
+		yield return new TestData<FlagsStringEnum>(FlagsStringEnum.A, ExpectedJsonSchema: """{"type":"string"}""");
```

Comment:
There are multiple flags-enum-to-string approaches:

- array of flags
- comma- or pipe-delimited

(My comment actually probably isn't a good idea since you can have 2^n options.)

Reply:
The converter uses comma separated. The number of valid inputs is actually infinite because for example `"A, A, A, A, A, A, A, A, A"` is accepted.